tests/telemetry/test_telemetry.py

```diff
@@ -634,6 +634,32 @@ def my_function(payload):
     )
 
 
+def test_log_call_add_payload_in_class_success(mock_telemetry):
+    _telemetry = telemetry.Telemetry(MOCK_API_KEY, "some-package", "0.14.0")
+    telemetry_my_class = _telemetry.create_group("TestClass")
+
+    class TestClass:
+        @telemetry_my_class.log_call("some-action", payload=True)
+        def my_function(self, payload, x, y):
+            payload["sum"] = x + y
+
+    test_class = TestClass()
+    test_class.my_function(1, 2)
```

Comment:
let's mak the function return some value and test it as well

Reply:
also add one test for fucntions with other signatures: no arguments, some keyword arguments, keyword-only args (e.g., `function(a, *, b, c)`, `function(*, a, b)`) - let's ensure all cases work

Reply:
Added more test cases in: [e13b976](https://github.com/ploomber/core/pull/45/commits/e13b976c5c6f0037d9107b8757f4e42d73a3be83)

Now cover: `no args`, `double asterisk`, `single asterisk`, `keyword_only w positional`, `keyword_only wo positional`